src/styles/card.css

```diff
@@ -0,0 +1,132 @@
+html {
+  --primary-color: #00acdc;
+  --main-text-color: #060b35;
+  --text-secondary-color: #616070;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--text-secondary-color);
+}
+
+body {
+  margin: 0;
+  --margin-right-left: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
What do you think if we moved this to the top?

Reply:
Owkay do you think it's best above body tag or html tag?

Reply:
Owkay. Noted thanx @leny-jeanbaptiste. Have moved it above html selector.